package.json

```diff
@@ -9,6 +9,7 @@
     "lint": "vue-cli-service lint"
   },
   "dependencies": {
+    "axios": "^0.21.4",
```

Comment:
Which component requires axios? We're currently using the browser native `fetch()` with polyfill and it would be great if we could stick with that (but it's not essential).

Reply:
download button; i did not have a lot of time for that, i plucked a snippet from SO to do that, but I agree it is clunky and should be improved. I will take a look.

Reply:
fixed it in next commit